subworkflows/local/db_check.nf

```diff
@@ -41,7 +41,17 @@ workflow DB_CHECK {
 
     //Filter the channel to run untar on DBs of tools actually using
     ch_input_untar = ch_dbs_for_untar.untar.dump()
-                    .filter {  params.run_kraken2 && it[0]['tool'] == 'kraken2' || params.run_centrifuge && it[0]['tool'] == 'centrifuge' || params.run_bracken && it[0]['tool'] == 'bracken' || params.run_kaiju && it[0]['tool'] == 'kaiju' || params.run_krakenuniq && it [0]['tool'] == 'krakenuniq' || params.run_malt && it[0]['tool'] == 'malt' || params.run_metaphlan3 && it[0]['tool'] == 'metaphlan3' }
+                    .filter {
```

Comment:
The comment above on line 42 is not clear to me. Maybe you could change it to:

> Filter the channel to untar only those databases that are selected to be run by the user.

Love the formatting! So much more legible. I have a cheeky idea to replace it with something simpler, though 😛 

```nextflow
.filter {
  params["run_${it[0]['tool']}"]
}
```

Reply:
well isn't that clever ;) , I'll finish my current local-test with this one, and then try adding your suggestion and see i it also works

Reply:
OK previous one works, will try now!